docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+
+Run the following from the repository root of your local clone:
+```bash
+pre-commit install
+```
+
+Now, a number of code linters and formatters will automatically run when you attempt to make a commit, which will keep your changes consistent with the rest of the codebase.
+
+#### 4. [Create a new branch](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/) to make the proposed changes
+Please consider using descriptive branch names. Some examples:
+
+- `<username>/<issue-identifier>` (`jsmith/fix-1234`)
+- `<username>/<brief-change-title>` (`jsmith/add-logging`, `jsmith/enh/add-logging`)
+
+Once you are satisfied with your local changes, [add, commit, and push](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository#adding-a-file-to-a-repository-using-the-command-line) them to your branch on your forked repository on GitHub. 
+
+#### 5. [Open a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+See the below sections for information on [how to submit your pull request](#pull-request-guidelines) and [what to expect in a pull request review](#pull-request-reviews).
+
+## Pull request guidelines
+
+When you first open a pull request, you should automatically see a template in the pull request body that looks something like [this](https://github.com/neurobagel/.github/blob/main/.github/pull_request_template.md?plain=1) that you can fill out.
+The template is designed to make it easier for maintainers to review your pull request, but feel free to add any additional information that you feel is useful or necessary.
+
+Pull request titles should begin with a descriptive prefix (e.g., "[ENH] Implement check for presence of a session ID column"):
+
+- `[ENH]:` Feature improvements or additions
+- `[REF]:` Refactoring existing code
+- `[TST]`: Updating or adding a test
+- `[CI]`: Automation-related changes
+- `[MNT]`: General maintenance not covered by `[REF]`, `[TST]`, or `[CI]`
+- `[INF]`: Software or graph infrastructure-related changes
+- `[FIX]`: Bug fixes
+- `[MODEL]`: Updates or changes related to the Neurobagel data model
+- `[DOC]`: Documentation-only changes to a code repo (READMEs, within-code documentation, etc.) 
+    - Exception: changes to the [`documentation`](https://github.com/neurobagel/documentation) repo should use one of the below PR prefixes instead of `[DOC]`
+
+If you are an external contributor, once your pull request is opened,
+we recommend that you [request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from a [Neurobagel core maintainer](http://127.0.0.1:8000/contributing/team/#developers) to ensure it gets reviewed in a timely manner.
```

Comment:
localhost seems to sneak into your links sometimes. copy/paste problem maybe?

Reply:
> localhost seems to sneak into your links sometimes. copy/paste problem maybe?

Heh, yep. I sometimes fill URLs in at the end, maybe too speedily. Looks like I missed the markdown link check here - will make sure to double check next time.

> more generally: do we want that? i.e. people requesting review directly? We should maybe add a bot to label and move the PR to the board. But it'd be tricky for external folks to know who to add here the page you linked includes many folks - and they may also be busy.

Yeah, good point. I'm okay with removing that recommendation for now until we have a better process. Sounds good?

Reply:
> more generally: do we want that? i.e. people requesting review directly? We should maybe add a bot to label and move the PR to the board. But it'd be tricky for external folks to know who to add here the page you linked includes many folks - and they may also be busy.

I second this.
We could use the `_community` label for now and update its description to not be a `BOT ONLY` or add another label for bot PRs.

Reply:
That solution sounds good @neurobagel/dev.

I'll separate the labelling business into a separate issue, and remove the recommendation for requesting a review for now. We can update this page once the labels are updated.

Reply:
See:
- https://github.com/orgs/neurobagel/projects/1/views/34?pane=issue&itemId=39843347